deploy/template.yaml

```diff
@@ -875,6 +875,56 @@ Resources:
         SSEEnabled: true
         SSEType: KMS
 
+  FE5XXErrorCriticalAlarm:
```

Comment:
I think this should have replaced/updated the original alarm `FE5XXErrorAlarm` https://github.com/govuk-one-login/ipv-cri-check-hmrc-front/blob/a1325e7be37ec0826c79f4eed3c8dd90a2a5d040/deploy/template.yaml#L928
Or the original needs downgrading to a warning alarm and removing from pager

Reply:
To me the new alarm is really different from the existing one. Originally I was going to replace it. But decided to leave it in the unlikely event of a higher than usual traffic suitation